docs/_data/nav-docs.yaml

```diff
@@ -10,7 +10,7 @@
     title: Create AWS stack
   - id: creating-a-vagrant-stack
     title: Create Vagrant stack
-  - id: connect-your-machine
+  - id: use-your-own-ide
```

Comment:
this needs to be same with https://github.com/sinan/koding/pull/4/files#diff-22ee1ec7cf01bce9b1cca40a10f6dbccR4